stdeb.cfg

```diff
@@ -0,0 +1,6 @@
+[colcon-meson]
+No-Python2:
+Depends3: meson (>= 0.60.0), python3-colcon-core (>= 0.10.0), python3-colcon-library-path
+Recommends3: ninja-build
```

Comment:
Is `ninja-build` used directly by `colcon-meson`? It's a hard dependency of the [meson](https://packages.debian.org/sid/meson) debian package and so if it's just there to support meson I think it can be dropped.

Reply:
No, it's only used by meson. The reason I added it to Recommends3 is related to the other. If meson is only recommended and installed externally via pip, then ninja has also to be installed manually since the meson pip package will not resolve the ninja package. If meson becomes a hard dependency, then we can remove the ninja dependency.

Reply:
Thanks for the info. I pushed [6ebacc8](https://github.com/colcon/colcon-meson/pull/8/commits/6ebacc81fb8c31216282db30194cf63df2a9b290) which removes this entirely since it's no longer useful for the debs.